README.md

```diff
@@ -15,11 +15,14 @@ operators in production.
   as any Kubernetes runtime object.
 - Managing [finalizers][finalizers] on reconciled objects, making sure the code
   is executed at least once for each create/delete/update event.
+- Trying to reduce number of deletion events after successful deletion event
+  reconciliation to avoid unnecessary, possibly expensive interactions with
+  third party systems.
```

Comment:
The sentence makes sense, but I had to attempt to read it 5 times until I got it. Maybe I am dumb. :D

Reply:
I changed it together with cache impl. Hopefully it's better. I'm open to suggestions.